test/integration/smoke.spec.ts

```diff
@@ -82,53 +34,29 @@ describe('Application launch', function foo() {
   });
 
   it('should create and open a new diagram by clicking on new diagram link', async () => {
-    await createAndOpenDiagram();
+    await testClient.createAndOpenNewDiagram();
 
     await testClient.assertNavbarTitleIs('Untitled-1');
     await testClient.assertWindowTitleIs('Design | BPMN Studio');
   });
 
-  it('should render a diagram correctly', async () => {
-    await createAndOpenDiagram();
-
-    await testClient.ensureVisible('[data-element-id="Collaboration_1cidyxu"]');
-  });
-
-  it('should select StartEvent after opening a diagram', async () => {
-    await createAndOpenDiagram();
-    await testClient.showPropertyPanel();
-
-    await testClient.assertSelectedBpmnElementHasName('Start Event');
-  });
-
-  it('should select element on click', async () => {
-    await createAndOpenDiagram();
-    await testClient.showPropertyPanel();
-
-    const elementName = 'End Event';
-
-    await testClient.rejectSelectedBpmnElementHasName(elementName);
-    await testClient.clickOnBpmnElementWithName(elementName);
-    await testClient.assertSelectedBpmnElementHasName(elementName);
-  });
-
   it('should create and open a second diagram', async () => {
-    await createAndOpenDiagram();
+    await testClient.createAndOpenNewDiagram();
 
     await testClient.assertNavbarTitleIs('Untitled-1');
     await testClient.assertWindowTitleIs('Design | BPMN Studio');
 
-    creatingFirstDiagram = false;
-    await createAndOpenDiagram();
+    testClient.creatingFirstDiagram = false;
+    await testClient.createAndOpenNewDiagram();
 
     await testClient.assertNavbarTitleIs('Untitled-2');
   });
 
   it('should open detail view', async () => {
```

Comment:
Zwecks einheitlicher Formulierung (die test cases weiter unten sind so formuliert. Und es liest sich einfach angenehmer).

```suggestion
  it('should open the detail view', async () => {
```